### ISSUE TITLE
Using Dimention blend only client side

### ISSUE THREAD

**Issue by JustJP2K** [Lauriethefish/BetterPortals#11]
Could you change how Biome Blend works? that way it only changes the textures client side but not actually changes the blocks. That way when the portal gets broken the blocks revert black to their original texture and dont stay that way. 
Would be a very nice and helpful touch

**Comment by Lauriethefish** [comment]
I like this idea, it's a little complex to implement though.
Would you need the exact blended blocks to stay the same when moving away and going back to the portal/relogging? If not, it's significantly easier to add.

May come in the future if that's fine, depends on if I think that it's worth it.
Nice to see that people actually use this feature - I wasn't sure who'd want it.

**Comment by JustJP2K** [comment]
Yeah its really neat. Sadly i cant use it since if players link the portal to the nether roof it also spawns bedrock into the overworld :(
Thats why making it client side would be nice

**Comment by Lauriethefish** [comment]
Ah that bug will be fixed. Didn't think about that when I added it last update xD

**Comment by JustJP2K** [comment]
ye :P
